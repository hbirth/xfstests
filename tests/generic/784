#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (c) 2023 Horst Birthelmer.  All Rights Reserved.
#
# FS QA Test 732
#
# test revalidate by underlying file change but with O_CREAT included.
# test dentry creation by the atomic revalidate 
#
. ./common/preamble
_begin_fstest auto

# Override the default cleanup function.
# _cleanup()
# {
# 	cd /
# 	rm -r -f $tmp.*
# }

# Import common functions.
# . ./common/filter


_supported_fs fuse
_require_test
_require_scratch

srcdir=/tmp/src
mkdir -p $srcdir

s_file=$srcdir/test_file
d_file=$SCRATCH_MNT/test_file

_scratch_mount "-o source=$srcdir"
echo "atod" > $d_file
cat $d_file
cat $s_file
rm -f $s_file
echo "btos" > $s_file
echo "ctod" >> $d_file
cat $d_file
rm -f $d_file

status=0
exit

