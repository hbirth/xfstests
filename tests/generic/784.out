QA output created by 732
